android/player/BUILD

```diff
@@ -16,7 +14,6 @@ kt_db_android_library(
     resources = main_resources,
     tags = ["maven_coordinates=com.intuit.playerui:android:aar:%s" % VERSION],
     visibility = ["//visibility:public"],
-    exports = main_exports,
```

Comment:
This should still use the `main_exports` if defined in `de(f|p)s.bzl` -- either add back, or flatten in the definition. I personally don't think the android player should be exporting the runtime.